dspace-server-webapp/src/main/java/org/dspace/app/rest/hdlresolver/HdlResolverDTO.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.hdlresolver;
```

Comment:
Why is this DTO in `dspace-server-webapp`?  It looks (to me) like this is the first DTO we are distributing in webapp directly.  Arguably, the DTO (along with the `HdlResolverService` and implementation should be in `dspace-api` instead...alongside all other DTOs.  Only the `RestController` looks like it's specific to the REST API.

It seems, to me, that this might be more appropriately moved to the existing `org.dspace.handle` package (in dspace-api), perhaps under a new subpath named `org.dspace.handle.hdlresolver`?   But, if I'm missing some reason why it should be in the REST API, let me know.